test/integration/history/test_spark_history_server.py

```diff
@@ -74,10 +83,11 @@ def test_history_server_with_expected_failure(mock_print, tag, role, image_uri,
         sagemaker_session=sagemaker_session,
     )
 
-    spark.start_history_server(spark_event_logs_s3_uri="invalids3uri")
+    with TestCase.assertLogs("sagemaker", level="ERROR") as cm:
+        spark.start_history_server(spark_event_logs_s3_uri="invalids3uri")
     response = _request_with_retry(HISTORY_SERVER_ENDPOINT, max_retries=5)
     assert response is None
-    mock_print.assert_called_with("History server failed to start. Please run 'docker logs history_server' to see logs")
+    assert "History server failed to start. Please run 'docker logs history_server' to see logs" in cm.output[0]
```

Comment:
So far we've just used `pytest`, not `unittest.TestCase` -- could you assert logs with pytest?

https://docs.pytest.org/en/stable/logging.html

Reply:
Done.